server/export_test.go

```diff
@@ -44,7 +44,6 @@ func TestExportCmd_ConsensusParams(t *testing.T) {
 		t.Fatalf("error unmarshaling exported genesis doc: %s", err)
 	}
 
-	require.Equal(t, genDoc.ConsensusParams.Block.TimeIotaMs, exportedGenDoc.ConsensusParams.Block.TimeIotaMs)
```

Comment:
Any reason we're deleting this test?

Reply:
TimeIotaMs has been deprecated. It is no longer used and has a hard coded value instead so I imagine it doesn't make too much sense to test it

Reply:
Makes sense!